### PR TITLE
#647 - document storage conf

### LINKED ISSUE #647: Expand readme
We lack some information in our readmes, at least: 
- repo permissions descriptions
- info about storages types and configuration

Let's add table of content to the readme after quick start section along with missing documentation.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #647 
I've [documented](https://github.com/olenagerasimova/artipie/tree/647-storage-conf) storage configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,45 @@ understand how Artipie is designed.
...
+For now, we support two storage types: file system and [S3](https://aws.amazon.com/s3/?nc1=h_ls) storages. 
+To configure file system storage it is enough to set the path where Artipie will store all the items:
+
+```yaml
+storage:
+  type: fs
+  path: /urs/local/aripie/data
+```
+
+S3 storage configuration requires specifying `backet` and `credentials`:
```
Comment (olegmoz) [inline README.md:93]: @olenagerasimova there is typo, `backet` should be `bucket`

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, it looks good. Please fix one typo and we are good to merge

**Comment by olenagerasimova** [comment]
@olegmoz thanks, corrected

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, merging

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#651` is not assigned, can't get performer

**Pull request merged**